src/app/modules/ix-tree/ix-tree-flattener.ts

```diff
@@ -0,0 +1,73 @@
+import { TreeControl } from '@angular/cdk/tree';
+import { Observable, take } from 'rxjs';
+
+export class IxTreeFlattener<T, F, K = F> {
```

Comment:
Generally speaking we've used `ix` prefixes to distinguish new things from things that may already be in the system with the same name. 
No need to use them for things that don't have existing equivalents.

Reply:
Yeah makes sense. I'll revisit it before merging.

Reply:
Reference to the ticket - https://ixsystems.atlassian.net/browse/NAS-119275